modules/plugin/geopkg/src/test/java/org/geotools/geopkg/GeoPkgDataStoreContentsOnlyTest.java

```diff
@@ -0,0 +1,38 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2022, Open Source Geospatial Foundation (OSGeo)
+ *
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License as published by the Free Software Foundation;
+ *    version 2.1 of the License.
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ */
+package org.geotools.geopkg;
+
+import java.sql.SQLException;
+import org.geotools.jdbc.JDBCBooleanOnlineTest;
+import org.geotools.jdbc.JDBCBooleanTestSetup;
+import org.junit.Test;
+
+public class GeoPkgDataStoreContentsOnlyTest extends JDBCBooleanOnlineTest {
```

Comment:
Good, but should not extend another test class with tests: you inherit those tests as well and they get run twice.
Instead:

```suggestion
public class GeoPkgDataStoreContentsOnlyTest extends JDBCTestSupport {
```

Reply:
Fixed.